src/helpers/generatePassword.js

```diff
@@ -0,0 +1,23 @@
+import bcrypt from 'bcrypt';
+/** Class representing a User services . */
+class Password {
+  /**
+   * assign password
+   * @param {string} data data to be checked against
+   */
+  constructor(data) {
```

Comment:
You can remove this constructor and create a static member function that will take a password as a parameter.